ingest-app/src/cmr/ingest/data/granule_bulk_update.clj

```diff
@@ -318,6 +324,8 @@
               (util/html-escape task-id))]))
   task-id)
 
+(declare task-completed?)
+#_{:clj-kondo/ignore [:unresolved-symbol]}
 (defn-timed task-completed?
```

Comment:
Do we need most of these functions are timed?

Reply:
there is a report out there.